keps/sig-node/4800-cpumanager-split-uncorecache/README.md

```diff
@@ -526,13 +526,18 @@ Reference CPUID info in podresources API to be able to verify assignment.
 
 ###### How can an operator determine if the feature is in use by workloads?
 
-Reference podresources API to determine CPU assignment and CacheID assignment.
+Reference podresources API to determine CPU assignment and CacheID assignment per container.
 Use proposed 'container_aligned_compute_resources_count' metric which reports the count of containers getting aligned compute resources.
 
 ###### How can someone using this feature know that it is working for their instance?
 
 Reference podresources API to determine CPU assignment.
 
+- [x] API .status
+  - Condition name: container_aligned_compute_resource_count
```

Comment:
technically speaking this is a metric one could use , not something in API .status field AFAICT

Reply:
updated